app-backend/src/utils/GitHelper.js

```diff
@@ -25,6 +25,8 @@ module.exports = {
 
         const USER = encodeURIComponent(username);
         const PASS = encodeURIComponent(pass);
+        
+        repo = repo.replace(/^(?:https?:\/\/)?/i, '');
```

Comment:
Thanks for the change, let's also add trim to cover more edge cases:
`repo = repo.trim().replace(/^(?:https?:\/\/)?/i, '');`